umh.docs.umh.app/content/en/docs/production-guide/upgrading/upgrading.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Management Console Upgrades
+content_type: task
+description: |
+  This page describes how to perform the upgrades that are available for the Management Console.
+weight: 2000
+---
+
+<!-- overview -->
+
+The Management Console simplifies the upgrade process for both the Management Companion and the UMH stack (not yet available).
+This page provides detailed instructions on executing these upgrades.
+
+## Management Companion
+
+Upgrade the Management Companion of your UMH instance to leverage the latest features and improvements.
+
+### {{% heading "prerequisites" %}}
+
+Before proceeding with the upgrade of the Companion, ensure that you have the following:
+
+- A functioning UMH instance, verified as "online" and in good health.
+- A reliable internet connection.
+- Familiarity with the changelog of the new version you are upgrading to, especially to identify any breaking changes or required manual interventions.
```

Comment:
This paragraph should be the first one after the overview, with the heading in H2